packages/video_player/video_player_android/lib/src/android_video_player.dart

```diff
@@ -174,3 +260,46 @@ class AndroidVideoPlayer extends VideoPlayerPlatform {
     );
   }
 }
+
+PlatformVideoViewType _platformVideoViewTypeFromVideoViewType(
+  VideoViewType viewType,
+) {
+  return switch (viewType) {
+    VideoViewType.textureView => PlatformVideoViewType.textureView,
+    VideoViewType.platformView => PlatformVideoViewType.platformView,
+  };
+}
+
+/// Base class representing the state of a video player view.
+@visibleForTesting
```

Comment:
Similar to comments above, make this private, and don't make it `@visibleForTesting`.

Let's observe the output instead of querying the internal state for testing.